apps/oc_chef_wm/itest/setup_helper.erl

```diff
@@ -105,3 +114,57 @@ needed_apps() ->
      oc_chef_authz,
      oc_chef_wm].
 
+%% @doc Does the setup common to all CT suites in this app:
+%% starts the server, create an org, and a client for it
+%% Expects the Config to define the following keys:
+%% org_name, org_authz_id, authz_id, client_name
+base_init_per_suite(Config0) ->
+    OrgName = get_config(org_name, Config0),
```

Comment:
Fair point - r perhaps make it something that has be done explicitly, as { defaults, [client, org, etc]}? Or would that just be  overengineering it...